src/Administration/Resources/app/administration/.eslintrc.js

```diff
@@ -256,6 +256,7 @@ module.exports = {
                 'sw-test-rules/await-async-functions': 'error',
                 'max-len': 0,
                 'sw-deprecation-rules/private-feature-declarations': 0,
+                'jest/no-truthy-falsy': 'error',
```

Comment:
Not particulalry a fan of just disallowing these without suggesting alternatives - are there maybe some rule we can add suggesting what want developers to use? Like [`prefer-strict-equal`](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-strict-equal.md) as an example

Reply:
We already have `prefer-to-be`, maybe that's not triggering correctly or needs some adjustments?